samcli/commands/init/__init__.py

```diff
@@ -273,9 +273,9 @@ def do_cli(
     image_bool = name and pt_explicit and base_image
     if location or zip_bool or image_bool:
         # need to turn app_template into a location before we generate
-        templates = InitTemplates(no_interactive)
+        templates = InitTemplates()
```

Comment:
Curious about these changes, --no-interactive ensures that we do not launch into a interactive experience at all, was this redundant code or something that has changed?

Reply:
It looks like the value was passed but was never used

Reply:
it was passed but never used